.gitignore

```diff
@@ -1,11 +1,13 @@
 # These are backup files generated by rustfmt
 **/*.rs.bk
 
-# required by nix
-.direnv/
-result/
+# Nix build output
+/result
+/result-lib
+/result-dev
+/result-man
 
-target/
```

Comment:
This one I rather keep because I use rio in a private fork that I have been working and target is independent there for multi levels

Reply:
Why would that be the case? The repository is a workspace, and thus, `target` should only appear at `/`.

Having a directory/module named `target` is perfectly reasonable, perhaps a compromise would be to add `!/src/**/target`?

Or, I'm a big fan of adding `.gitignore` in directories which need special handling (for offline work, for example).